packages/config/config/devices/0x014f/lbr30z-1.json

```
@@ -16,38 +16,15 @@
 	"paramInformation": [
 		{
 			"#": "1",
-			"label": "Dim Level Memory",
-			"valueSize": 1,
-			"defaultValue": 0,
-			"allowManualEntry": false,
-			"options": [
-				{
-					"label": "Disable dim level memory",
-					"value": 0
-				},
-				{
-					"label": "Enable dim level memory",
-					"value": 1
-				}
-			]
+			"$import": "templates/config_template.json#init_brightness"
 		},
 		{
 			"#": "9",
-			"label": "Dim / Bright Step Level",
-			"description": "How much the brightness will change with each dimming step",
-			"valueSize": 1,
-			"minValue": 1,
-			"maxValue": 99,
-			"defaultValue": 1
+			"$import": "~/templates/master_template.json#dim_rate"
```

Shouldn't this be the dim step instead?
```suggestion
			"$import": "templates/config_template.json#dim_step_zwave"
```

---

Please double-check the other files too if they need to be adjusted